stories/index.js

```diff
@@ -666,6 +664,69 @@ storiesOf('TableHeading', module)
 
 storiesOf('Table', module)
   .add('base table', () => {
+    const noResults = props => (
+      <p>Nothing!</p>
+    );
+
+    return (
+      <Table
+        NoResults={noResults}
+      />
+    );
+  })
+
+  .add('empty with columns', () => {
+    const components = {
+      Table: ({ TableHeading, TableBody, NoResults, style, visibleRows }) => (
+        <table style={style}>
+          <TableHeading />
+          { visibleRows ? (TableBody && <TableBody />) : (NoResults && <NoResults />) }
+        </table>
+      ),
+      NoResultsContainer: compose(
```

Comment:
The main reason for making our container components higher-order components rather than wrappers is that we are assuming that they may be composed via plugins, etc. 

There may be a better / clearer way to do this moving forward but that was the initial intent (if I understand correctly :+1:)